src/client/per_process.cpp

```diff
@@ -772,3 +811,43 @@ void TrClientContextPerProcess::onListenMediaEvent(media_comm::TrMediaCommandMes
     fprintf(stderr, "ClientContext(%d) received an unknown media event message\n", id);
   }
 }
+
+bool TrClientContextPerProcess::sendInspectorResponse(inspector_comm::TrInspectorCommandBase &response)
+{
+  assert(inspectorChanSender != nullptr);
+  return inspectorChanSender->sendCommand(response);
+}
+
+void TrClientContextPerProcess::onInspectorCommand(inspector_comm::TrInspectorCommandMessage &commandMessage)
+{
+  auto messageType = commandMessage.getType();
+
+  if (messageType == inspector_comm::TrInspectorCommandType::CdpRequest)
+  {
+    auto cdpRequest = inspector_comm::TrInspectorCommandBase::CreateFromMessage<inspector_comm::TrCdpRequest>(commandMessage);
```

Comment:
You need to get the domain and method here, then dispatch the request to corresponding content-specific domain handler at client/inspector.